googleapis/google-cloud-node
# migrate code from googleapis/nodejs-deploy

- feat: initial stub of library
- feat: initial generation of library (#1)
- chore: release 0.2.0 (#3)
- chore(deps): update dependency mocha to v9 (#4)
- chore(deps): update dependency sinon to v11 (#5)
- docs: update reference documentation url (#8)
- build(node): update deps used during postprocessing (#1243) (#6)
- docs: fix docstring formatting (#9)
- chore: fix the wrong post processor image (#11)
- chore(deps): update dependency @types/node to v16 (#10)
- chore(deps): update dependency sinon to v12 (#12)
- docs(samples): add example tags to generated samples (#13)
- build: add generated samples to .eslintignore (#14)
- docs(node): support "stable"/"preview" release level (#1312) (#17)
- chore: add api_shortname and library_type to repo metadata (#16)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#19)
- test(nodejs): remove 15 add 16 (#1322) (#21)
- build: update copyright year to 2022 (#23)
- chore: add codeowners (#25)
- chore: update v2.12.0 gapic-generator-typescript (#26)
- docs(nodejs): version support policy edits (#1346) (#30)
- chore(deps): update dependency sinon to v13 (#28)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#31)
- chore: update v2.14.2 gapic-generator-typescript (#35)
- feat: promote to stable (#36)
- chore(main): release 1.0.0 (#37)
- docs: Cloud Deploy API Platform Logging documentation (#39)
- build(node): update client library version in samples metadata (#1356) (#45)
- chore(deps): update dependency sinon to v14 (#48)
- build!: update library to use Node 12 (#50)
- chore(main): release 2.0.0 (#51)
- chore(deps): update dependency jsdoc-fresh to v2 (#53)
- chore(deps): update dependency jsdoc-region-tag to v2 (#54)
- feat: support regapic LRO (#55)
- chore(main): release 2.1.0 (#56)
- chore(deps): update dependency linkinator to v4 (#58)
- fix: better support for fallback mode (#59)
- fix: change import long to require (#60)
- chore(main): release 2.1.1 (#61)
- fix: do not import the whole google-gax from proto JS (#1553) (#63)
- fix: allow passing gax instance to client constructor (#64)
- fix: preserve default values in x-goog-request-params header (#66)
- build: use fully qualified request type name in integration tests (#68)
- chore(main): release 2.1.2 (#65)
- chore: override API mixins when needed (#70)
- chore(main): release 2.2.0 (#71)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
